### DISCUSSION TITLE
restore deleted person

### DISCUSSION THREAD

**Discussion by thomas64600** [nextcloud/recognize#946] [Q&A] [answered]
Hello,
I deleted a person by mistake, I want to know if it is possible to restore it from the application?
Thanks. 
Thomas

**Comment by marcelklehr** [comment, marked as answer]
Hey @thomas64600 

This is not possible, currently. You can only try running recognize:reset-face-clusters and re-running recognize:cluster-faces (see recognize admin panel) to recreate the original clusters, but then any manual modifications are lost.

**Reply by thomas64600** [reply]
Hello @marcelklehr , 
thank you for this information, it's a shame that the clusters are deleted in the database, it would be interesting to have a boolean to show/hide them.

**Reply by marcelklehr** [reply]
What would be your use case for this? Could you describe your situation a little? :)
